.github/workflows/standard-go-test.yml

```diff
@@ -8,5 +8,5 @@ on:
 jobs:
   call-standard-test:
     name: Test
-    uses: gobuffalo/.github/.github/workflows/go-test.yml@v1
+    uses: gobuffalo/.github/.github/workflows/go-test.yml@v1.6
```

Comment:
After the next core release I am preparing nowadays, I will update the tag `v1` to point `v1.6` so all packages will use the versions! Thanks!